src/Simplic.Studio.Tenant.UI/OrganizationViewModel.cs

```diff
@@ -0,0 +1,264 @@
+﻿using Simplic.TenantSystem;
+using Simplic.UI.MVC;
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+
+namespace Simplic.Studio.TenantSystem.UI
+{
+    /// <summary>
+    /// OrganizationViewModel to handle tenant data.
+    /// </summary>
+    public class OrganizationViewModel : ViewModelBase
+    {
+        private IOrganizationService organizationService = CommonServiceLocator.ServiceLocator.Current.GetInstance<IOrganizationService>();
+        private ObservableCollection<Organization> organizations;
+        public int subOrganizationCount;
+        public IList<Guid> subOrganizations;
+        public Guid? cloudOrganizationId;
+        private Organization model;
+        private bool isCreate;
+
+        /// <summary>
+        /// Default constructor for the view model
+        /// </summary>
+        public OrganizationViewModel()
+        {
+            Organizations = new ObservableCollection<Organization>();
+
+            foreach (var organization in organizationService.GetAll())
+            {
+                Organizations.Add(organization);
+            }
+
+            this.IsCreate = true;
+        }
+
+        /// <summary>
+        /// Constructor for the view model
+        /// </summary>
+        /// <param name="id">Tenant ID</param>
+        /// <param name="name">Tenant name</param>
+        /// <param name="matchCode">Tenant match code</param>
+        /// <param name="subOrganizationCount">Tenant suborganizations count</param>
+        /// <param name="subOrganizations">Tenant suborganizations IDs collection</param>
+        /// <param name="cloudOrganizationId">Tenant cloud organization id</param>
+        /// <param name="isActive">Tenant active flag</param>
+        /// <param name="isGroup">Tenant group flag</param>
+        public OrganizationViewModel(Guid id, string name, string matchCode, int subOrganizationCount,
```

Comment:
First of all this constructor is unused outside of the call to the constructor beneath and we dont follow the pattern of adding constructor parameters to initialize a new instance but try to use default values when the value is not required in the constructor. And none of the values here are required in the constructor since the only caller will overwrite the model instance anyway.